import java.lang.Exception;

class M13
                                                    // arithmatic exception program
{

 public static void main(String[] args)
 {
  // String str = "Sagar";
   int str = 123;  

        try{

                                        
  int a = str;                    //    is line m exception aya
  System.out.println(a);
  
   

 }

catcnumberxception n)                        //  isme abnormal tarike se program terminate hoga 
                                                    //  kuki isme arithmatic nhi h to = koi bhi 
                                                    //  function nhi chalega
{


System.out.println(n);

}


   System.out.println("main method ended");


}}

